GitCommands/Git/GitModule.cs

```diff
@@ -1522,9 +1522,14 @@ private ArgumentString GetFetchArgs(string? remote, string? remoteBranch, string
                 }
             }
 
+            bool remoteJobs = string.IsNullOrWhiteSpace(EffectiveConfigFile.GetValue("fetch.parallel"));
+            bool submoduleJobs = string.IsNullOrWhiteSpace(EffectiveConfigFile.GetValue("submodule.fetchJobs"));
+
             // TODO return ArgumentBuilder and add special case ArgumentBuilder.Add(ArgumentBuilder childBuilder)
             return new ArgumentBuilder
             {
+                { remoteJobs && string.IsNullOrWhiteSpace(branchArguments), "--multiple" },
```

Comment:
The Git documentation mentions this option. Though it seems to be implicit, i.e. not necessary.

Reply:
Removed --multiple